.circleci/config.yml

```diff
@@ -69,3 +69,52 @@ jobs:
       - store_artifacts:
           path: test-results
           destination: tr1
+
+
+  deploy_pypi:
+    docker:
+      - image: circleci/python:3.7
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      - run:
+          name: configure pypirc
+          command: |
+            echo -e "[pypi]" >> ~/.pypirc
+            echo -e "username = $PYPI_USERNAME" >> ~/.pypirc
+            echo -e "password = $PYPI_PASSWORD" >> ~/.pypirc
+
+      - run:
+          name: setup_venv
+          command: |
+            python3 -m venv venv
+            . venv/bin/activate
+            pip install -U twine setuptools wheel
+
+      - run:
+          name: build wheels
+          command: |
+            . venv/bin/activate
+            python setup.py sdist
+            python setup.py bdist_wheel
+
+      - run:
+          name: deploy
+          command: |
+            . venv/bin/activate
+            twine upload dist/* --verbose --config-file ~/.pypirc
+
+workflows:
+  version: 2
+  build_and_deploy:
+    jobs:
+      - build
+      - deploy_pypi:
+          requires:
+            - build
+          filters:
+            branches:
+              only: 326/deploy-pypi
```

Comment:
might go ahead and just change this to its intended value now that you've verified its working

Reply:
change to "release" branch as Sergey proposed.

Reply:
wait - so there is a branch called release and it is going to be our mainline?

Reply:
added the parts to accept the git tag v###. thanks.